spec/system/group_spec.rb

```diff
@@ -36,8 +36,11 @@
     click_button "+ Add Members"
     execute_script "document.getElementById('addmemberModal').style.display='block'"
     execute_script "document.getElementById('addmemberModal').style.opacity=1"
-    fill_in "group_email_input", with: @user2.email
-    fill_in "group_email_input", with: " "
+    execute_script "var new_email = document.createElement('option')
```

Comment:
This is a hack, we should be able to do this without this hack, I am merging this now to unlblock CI. Could you create a followup issue to simplify this ?

Reply:
yaa sure